dnf5daemon-client/commands/group/group_list.hpp

```diff
@@ -22,20 +22,56 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 
 #include "commands/command.hpp"
 
+#include <libdnf5-cli/session.hpp>
 #include <libdnf5/conf/option_enum.hpp>
+#include <libdnf5/utils/bgettext/bgettext-lib.h>
 
 #include <memory>
 #include <vector>
 
 namespace dnfdaemon::client {
 
+class GroupAvailableOption : public libdnf5::cli::session::BoolOption {
+public:
+    explicit GroupAvailableOption(libdnf5::cli::session::Command & command)
+        : BoolOption(command, "available", '\0', _("Show only available groups."), false) {}
+};
+
+class GroupInstalledOption : public libdnf5::cli::session::BoolOption {
+public:
+    explicit GroupInstalledOption(libdnf5::cli::session::Command & command)
+        : BoolOption(command, "installed", '\0', _("Show only installed groups."), false) {}
+};
+
+class GroupHiddenOption : public libdnf5::cli::session::BoolOption {
+public:
+    explicit GroupHiddenOption(libdnf5::cli::session::Command & command)
+        : BoolOption(command, "hidden", '\0', _("Show also hidden groups."), false) {}
+};
+
+class GroupContainPkgsOption : public libdnf5::cli::session::AppendStringListOption {
+public:
+    explicit GroupContainPkgsOption(libdnf5::cli::session::Command & command)
+        : AppendStringListOption(
+              command,
+              "contains-pkgs",
+              '\0',
+              _("Show only groups containing packages with specified names. List option  , supports globs."),
```

Comment:
There are two extra spaces in the description.

Reply:
Thanks.